packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -115,6 +115,7 @@ are supported:
 * [`Succeed`](#succeed)
 * [`Fail`](#fail)
 * [`Map`](#map)
+* [`Custom`](#custom)
```

Comment:
CustomState?

Reply:
yep, good catch!

I honestly went back and forth between `Custom` and `CustomState` - although the rest of the states aren't suffixed, i went with it because it felt really weird to do `new Custom`